src/backend/src/prisma/schema.prisma

```diff
@@ -698,7 +698,7 @@ model Project_Proposed_Changes {
   teams                    Team[]               @relation(name: "proposedProjectTeams")
   wbsProposedChanges       Wbs_Proposed_Changes @relation(name: "projectProposedChanges", fields: [wbsProposedChangesId], references: [wbsProposedChangesId])
   wbsProposedChangesId     String               @unique
-  newProject               Boolean
+  carNumber                Int?
```

Comment:
Mmmmm this is making me realize we should have organized our proposed changes models differently but thats an issue for another time

Just a note to self